eng/pipelines/coreclr/templates/superpmi-collect-job.yml

```diff
@@ -33,7 +37,16 @@ jobs:
      - ${{ if eq(parameters.collectionName, 'coreclr_tests') }}:
         - 'coreclr_common_test_build_p1_AnyOS_AnyCPU_${{parameters.buildConfig }}'
 
-    variables: ${{ parameters.variables }}
+    variables:
+
+    - name: crossArg
+      value: ''
+    - ${{ if eq(parameters.crossBuild, true) }}:
+      - name: crossArg
+        value: '-cross'
+
+    - ${{ each variable in parameters.variables }}:
```

Comment:
Is this needed so it can get the `crossArg` variable value?

Reply:
Basically, we need to inherit all the passed in variables as well as set `crossArg`. This is one way to do that.